backend/src/libs/types/repository.type.ts

```diff
@@ -3,7 +3,7 @@ type Repository<T = unknown> = {
   findAll(): Promise<T[]>;
   create(payload: unknown): Promise<T>;
   update(payload: unknown): Promise<T>;
-  delete(): Promise<boolean>;
+  delete(id: number): Promise<number | boolean>;
```

Comment:
It should be only one type for returning

Reply:
@So, should we return back (as it was in the beginning) in our delete repository to this to return boolean everywhere?
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/102380564/7a39bd3b-759a-4186-a27b-a2b1473db8e7)

Reply:
So I changed it to have boolean everywhere

Reply:
Let's delete in repository level will return number)

Reply:
Ok